_topic_maps/_topic_map.yml

```diff
@@ -2229,6 +2229,14 @@ Topics:
   - Name: Placing pods on specific nodes using node selectors
     File: nodes-pods-node-selectors
     Distros: openshift-enterprise,openshift-origin
+  Name: Using run-once pods
```

Comment:
I'd recommend naming this "Run Once Duration Override Operator". Our directories are usually nouns